docs/pages/blog/introducing-pigment-css.md

```diff
@@ -0,0 +1,114 @@
+_We intend for Pigment CSS to be a viable solution for the React ecosystem as a whole, but you can understand why we're prioritizing Material UI during early development._
+
+### Better performance
+
+When comparing the same Material UI app built with Next.js and either Emotion or Pigment CSS, the latter led to:
+
+- 20% reduction in first load JavaScript (104 kB vs. 131 kB)
+- 9% decrease in First Contentful Paint (455 ms vs. 503 ms)
+- 15% reduction in Time To First Byte
+- 7.5% reduction in page HTML (14.7 kB vs. 15.9 kB)
+
+### Familiar developer experience
+
+For developers migrating from Emotion or styled-components, you're probably already familiar with the most common patterns employed by Pigment CSS.
+`styled()` and `css()` are the two main functions used to define styles, and they mostly work the same as you'd expect them to (with some notable differences due to the nature of build-time CSS-in-JS—see [Coming from Emotion or styled-components](https://github.com/mui/pigment-css/tree/master?tab=readme-ov-file#coming-from-emotion-or-styled-components) for details).
+
+We've also ported over the `sx` prop from MUI System, so you can still define styles directly in a given component, but now it's much more performant than before.
+And in Pigment CSS we've extended support for `sx` to include _all_ DOM nodes—not just Material UI components—so you don't need to wrap a simple `<div>` or `<span>` with a Box component to apply theme styles to it.
+
+### Future-proof solution
+
+Though we're still quite early in the RSC era, it seems inevitable that the React ecosystem as a whole will converge on this new paradigm over time.
+Next.js gave us our first glimpse with the App Router; RedwoodJS [recently released their own implementation](https://redwoodjs.com/blog/rsc-now-in-redwoodjs); and many other frameworks and meta-frameworks are currently working out POCs and RFCs to catch up.
+Regardless of how quickly Server Components catch on among developers, it's clear that library maintainers must now support [the two Reacts](https://overreacted.io/the-two-reacts/) (client-side and server-side) to stay relevant into the future.
+
+Pigment CSS, then, is yet another bet from MUI on the longevity and sustainability of the React ecosystem—and a promise that we'll continue to innovate in this space for years to come.
+
+And perhaps most importantly: because Pigment CSS is maintained by the same folks behind Material UI, we'll have a lot more control over how the tool evolves over time to continue to meet our users' needs.
+In a perfect world, this would be the last time you'd ever have to migrate your Material UI app to a new style engine.
+We'll do our best to make that a reality. 🤞
+
+## What's next
+
+Pigment CSS is currently in the early alpha stage of development—the plan is to have a stable version ready to release alongside Material UI v6 later this year.
```

Comment:
Maybe worth rephrasing given today's discussion about Material's v6 launching with Pigment still in v0? 😬

Reply:
```suggestion
Pigment CSS is currently in the early alpha stage of development—the plan is to have a fully featured version ready to release alongside Material UI v6 later this year.
```